src/input_viewer.cpp

```diff
@@ -23,16 +23,16 @@ namespace InputViewer {
         float dy2 = -0.7071f;
         float tmp;
         Draw::begin(8, GX_TRIANGLEFAN);
-            for (uint8_t i = 0; i < 4; ++i) {
-                Draw::add_vertex(color, {pos.x + s2 * dx1, pos.y + s2 * dy1}, {0.5f + dx1, 0.5f + dy1});
-                Draw::add_vertex(color, {pos.x + s2 * dx2, pos.y + s2 * dy2}, {0.5f + dx2, 0.5f + dy2});
-                tmp = dx1;
-                dx1 = -dy1;
-                dy1 = tmp;
-                tmp = dx2;
-                dx2 = -dy2;
-                dy2 = tmp;
-            }
+        for (uint8_t i = 0; i < 4; ++i) {
+            Draw::add_vertex(color, {pos.x + s2 * dx1, pos.y + s2 * dy1}, {0.5f + dx1, 0.5f + dy1});
+            Draw::add_vertex(color, {pos.x + s2 * dx2, pos.y + s2 * dy2}, {0.5f + dx2, 0.5f + dy2});
+            tmp = dx1;
+            dx1 = -dy1;
+            dy1 = tmp;
+            tmp = dx2;
+            dx2 = -dy2;
+            dy2 = tmp;
+        }
```

Comment:
The indentation with the Draw functions are intended. It's to signal visually that everything between `begin` and `end` have to be encapsulated between them.